tests/test_shell.py

```diff
@@ -255,15 +255,15 @@ def test_find(self):
                 self.call(
                     "python -m signac find ".split() + ['{"doc.a": ' + str(i) + "}"]
                 ).strip()
-                == [job.id for job in project.find_jobs(doc_filter={"a": i})][0]
+                == [job.id for job in project.find_jobs({"doc.a": i})][0]
```

Comment:
Please duplicate the test instead of altering it to ensure that we do not introduce a regression in future releases.

Reply:
See conversation below -- this is a test of the CLI. The `find_jobs` call is only for comparison. I don't think duplication is necessary.

Reply:
I think this is fine.

Reply:
Agreed and sorry for not responding earlier. Somehow I missed this. All good.